draft-js-mention-plugin/src/MentionSuggestions/index.js

```diff
@@ -49,14 +49,6 @@ export class MentionSuggestions extends Component {
     this.props.callbacks.onChange = this.onEditorStateChange;
   }
 
-  componentWillReceiveProps(nextProps) {
-    if (nextProps.suggestions.size === 0 && this.state.isActive) {
-      this.closeDropdown();
-    } else if (nextProps.suggestions.size > 0 && !nextProps.suggestions.equals(this.props.suggestions) && !this.state.isActive) {
-      this.openDropdown();
-    }
-  }
-
```

Comment:
this can't be removed I'm afraid - and it's out of scope for this pr really

Reply:
This is how I resolved the arrow key issue for myself, but I understand that this probably serves a purpose. The issue I was running into was `this.state.isActive` was `true` even though the suggestions wasn't open, therefore `keyboardEvent.preventDefault()` below was being called and the up/down arrow key events were being intercepted by this. What is the purpose of opening the suggestions when `it's not active and the suggestions is not empty and the suggestions have changed`?

Reply:
All fair points. I'm afraid that removing this causes other things to break. I can't remember off the top of my head but it's there for a reason, even though it's awkward. A bunch of logic will need to be rewritten to fix these issues :/